src/Items/ItemDefaultItem.h

```diff
@@ -0,0 +1,12 @@
+//
+// Created by marc_ on 26.11.2021.
+//
+
```

Comment:
Hopefully Git keeps this information without the assistance of comments!

Reply:
My IDE being "helpful"